src/bin/nydus-image/main.rs

```diff
@@ -397,7 +397,25 @@ fn prepare_cmd_args(bti_string: &'static str) -> App {
                             .required(false),
                     )
                     .arg(arg_output_json.clone())
-            )
+                )
+                .subcommand(
+                    App::new("generate")
```

Comment:
Good job. Please add smoke [tests](https://github.com/dragonflyoss/nydus/tree/master/smoke/tests) which cover changes for user. It's sometimes more useful than doc for developers.